### ISSUE TITLE
Instruction text consistency

### ISSUE THREAD

**Issue by mgwalker** [18F/cms-hitech-apd#1088]
Use this structure for instructional text in the yaml and throughout the app:

```yaml
instruction:
  heading:(h3) BLAH BLAH!
  short: (bold) blah blah
  detail:(regular text)  >-
    blah blah blah blah blah
    blah blah blah blah blah
    blah blah blah blah blah    
  helpText: (thing that appears in blue box)
```

We need to preserve the option in the code to switch the order of short/detail to accommodate the standards and conditions section.

### This task is done when...
 - [ ] all the instructional text in the yaml files has been reformatted
 - [ ] the app code uses the new format throughout

**Comment by jeromeleecms** [comment]
What's best practice for documenting this format for future generations? Is this documented in comments within the YAML file, go into Technical documentation, deserve its own page in the future style guide, etc...?

**Comment by mgwalker** [comment]
One cool thing about YAML is that we ***can*** have comments.  I think some kind of standalone thing makes more sense, though.  A place where we can list the textual components that the app is aware of and the structure it expects from the YAML, for example.  🤔

http://storybook.hitech.eapd.cms.gov.s3-website.us-east-2.amazonaws.com/

**Comment by mgwalker** [comment]
I'm curious about the logistics of making this change, because it seems like it could need considerable coordination between content and dev, since changing the keys in the YAML files will necessitate code changes.

I think the biggest two pieces for me are:
- renaming the `reminder` key to `helpText` (which doesn't actually feel all that problematic, except...)
- what to do with everything that's currently stored as `helpText`

My sense is that once we have a good strategy for how to tackle those two, dev can hop into code and make a lot of it happen, with some occasional pairing with design to iron out some weird edge cases.  Then, once the app is prepared to handle all the new content components, it should be easier for @jameshupp to cleanup any booboos we introduced and make everything just right.

**Comment by mgwalker** [comment]
@jameshupp I didn't mean to make any suggestions.  I don't have any reservations about you making structural changes as long as you loop in dev so we can do the associated code changes, especially around this issue since we've already talked about it.  I just wanted to remark that this issue hints at a fairly significant amount of coordination!  😄

**Comment by jeromeleecms** [comment]
Is the current state of the YAML file good enough for the MVP? Or does this revised formatting provide some real solid (aka save time down the road and/or other benefits) for the MVP?

**Comment by jameshupp** [comment]
Sorry, @jeromeleecms, by "current state of the YAML" do you mean _structure_ of it or _language_ in it? The actual language shouldn't block anything as long as the shape is manageable. I'm still a little confused about how to do make the structural adjustments. Need to pair with someone for a few minutes to understand better.

**Comment by jeromeleecms** [comment]
@jameshupp I was thinking structure. I.e. does the structure actually help make the development / presentation of the text easier?

**Comment by jameshupp** [comment]
Work on #1178 has revealed some YAML elements we haven't accounted for here that are doing similar work. For example: `milestoneHelpText`. Seems like it might be a one-off and it's easy to cover with the schema described at the top of this issue. I'm going to compile a list of them in this comment, and when I'm done with the rest of 1178 I'll come back here to see if it's as complex as it might be. If it's as simple as `milestoneHelpText` looks to be we'll have no problem and I can quickly swap those out alongside the rest.

**Comment by jameshupp** [comment]
Just as a note to self: `heading` is also not making a lot of appearances in my output and I'm starting to wonder if I've misplaced the idea for how we wanted to use that.

**Comment by jameshupp** [comment]
Following up on the earlier comment, here's why I left the ones in place that I left in place:
1. `milestoneHelpText` doesn't have a separate `instruction` block. It seems like, without that, there's a chance that two different `short` instructions would be confused with each other. Without that, they're in the same place in the hierarchy. Would like to pair on this to see if there's a good way to resolve that.

1. `helpText` outside the accordion: left in place because we don't have an alternative designation. In pairing, I will clarify what we can do about this.

1. the long list under `headers`: this is just a list in the yaml. Not sure what it would do to try to convert these.

On the first and third, I feel uncomfortable with the orphaned nature of these yaml elements. The second is fine; I just want a better label.

**Comment by jameshupp** [comment]
1 and 3 are now up to date. 2 is held for a separate issue, which I'll write shortly.

**Comment by mgwalker** [comment]
Is this one addressed by #1299 now, or is there more that needs doing?
